datasets/wit/wit.py

```diff
@@ -0,0 +1,99 @@
+"""
+_HOMEPAGE = "https://github.com/google-research-datasets/wit"
+_URL = "https://storage.googleapis.com/huggingface-nlp/datasets/wit/"
+_URLS = {"train": [_URL + f"part-{'%05d' % i}-48a6f07e-bb86-4735-aac7-883349f41a28-c000.json.gz" for i in range(400)]}
+
+
+class Wit(datasets.GeneratorBasedBuilder):
+    """WIT: Wikipedia-based Image Text Dataset for Multimodal Multilingual Machine Learning"""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=Features(
+                {
+                    "b64_bytes": Value("string"),
+                    "embedding": Sequence(Value("float64")),
+                    "image_url": Value("string"),
+                    "metadata_url": Value("string"),
+                    "original_height": Value("int32"),
+                    "original_width": Value("int32"),
+                    "mime_type": Value("string"),
+                    "caption_attribution_description": Value("string"),
+                    "wit_features": Sequence(
+                        {
+                            "language": Value("string"),
+                            "page_url": Value("string"),
+                            "attribution_passes_lang_id": Value("string"),
+                            "caption_alt_text_description": Value("string"),
+                            "caption_reference_description": Value("string"),
+                            "caption_title_and_reference_description": Value("string"),
+                            "context_page_description": Value("string"),
+                            "context_section_description": Value("string"),
+                            "hierarchical_section_title": Value("string"),
+                            "is_main_image": Value("string"),
+                            "page_changed_recently": Value("string"),
+                            "page_title": Value("string"),
+                            "section_title": Value("string"),
+                        }
+                    ),
+                }
+            ),
+            homepage=_HOMEPAGE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager):
+        """Returns SplitGenerators."""
+        urls_to_download = _URLS
+        downloaded_files = dl_manager.download_and_extract(urls_to_download)
+        return [
+            datasets.SplitGenerator(name=datasets.Split.TRAIN, gen_kwargs={"filepaths": downloaded_files["train"]}),
+        ]
+
+    def _generate_examples(self, filepaths):
+        """Yields examples."""
+        wit_feature_names = self.info.features["wit_features"].feature.keys()
+        example_id = 0
+        for filepath in filepaths:
+            with gzip.open(filepath, mode="rt", encoding="utf-8") as f:
+                for i, line in enumerate(f):
+                    line = line.strip()
+                    row_data = json.loads(line, encoding="utf-8")
```

Comment:
Hi. I got an error when trying with Python 3.9, due to the use of the `encoding` argument

> Changed in version 3.9: The keyword argument encoding has been removed.

(https://docs.python.org/3/library/json.html)

Reply:
@severo I looked into this and actually had a really annoying time trying to make something work. Since the CI isn't running 3.9, it seems we do not technically support it (also we only mention 3.6 + 3.7 in `setup.py`), but I'm sure there's a way for us to make this work (it just won't be testable). 

Would love some help if you know what to do 😄 .

Reply:
I have no idea... Maybe @lhoestq or @albertvillanova ?

Reply:
We could add CI workflows for other python versions if necessary. Currently we have Linux+python3.7 and Windows+python3.6

Reply:
Or feel free to test it locally if it's just a matter for this specific line

Reply:
feel free to remove the `encoding` argument here :)
```suggestion
                    row_data = json.loads(line)
```